CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v6.0.0(unreleased)
```

Comment:
Nice, just drop this unreleased and bump up versions as I'm assuming this will be released right away to be used with Chef.